Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -200,30 +211,60 @@ private void Update()
                     {
                         rightPoint.transform.position = GetContactForHand(Handedness.Right);
                     }
+
+                    
                 }
             }
         }
         #endregion MonoBehaviour Handlers
 
 
         #region Private Methods
+        private bool TryGetMRControllerRayPoint(HandPanData data, out Vector3 rayPoint)
+        {
+            if (data.currentController.InputSource.SourceName.Contains("Mixed Reality Controller"))
+            {
+                if (!(data.currentController.InputSource.Pointers[0] is GGVPointer))
```

Comment:
What if the GGVPointer exists but is not the first pointer in the list?

Reply:
isnt the first pointer in the list the one that is associated with the event?

Reply:
ill make this switch :)

Reply:
where are these types defined?

Reply:
oops missed one